spec/mail/message_spec.rb

```diff
@@ -1403,6 +1403,23 @@ def message_headers_should_match(message, other)
 
       describe "content-transfer-encoding" do
 
+        # https://www.ietf.org/rfc/rfc2046.txt
+        # No encoding other than "7bit", "8bit", or "binary" is permitted for
+        # the body of a "message/rfc822" entity.
+        it "rfc2046" do
+          mail = Mail.new
+          mail.body << Mail::Part.new.tap do |part|
+            part.header[:content_disposition] = 'attachment; filename="test.eml"'
```

Comment:
We have content disposition API available here. Do we need to set the header directly?